akka-cluster/src/test/scala/akka/cluster/protobuf/ClusterMessageSerializerSpec.scala

```diff
@@ -86,7 +86,7 @@ class ClusterMessageSerializerSpec extends AkkaSpec(
           totalInstances = 2,
           maxInstancesPerNode = 5,
           allowLocalRoutees = true,
-          useRole = Some("Richard, Duke of Gloucester")
+          useRoles = Set("Richard, Duke of Gloucester")
```

Comment:
Cover no roles and multiple roles as well?

It would also be good to have backwards bin-comp test coverage, deserialising an old message, you can see how to do that in https://github.com/akka/akka/blob/3643f18ded0cf88dfc098d707512b762fd1ff0f7/akka-remote/src/test/scala/akka/remote/serialization/DaemonMsgCreateSerializerSpec.scala#L108